src/taxes/income-tax.ts

```diff
@@ -498,12 +499,27 @@ function getSurtaxRates(code: ProvinceCode | FederalCode): Rate[] {
     return TAX_BRACKETS[code].SURTAX_RATES;
 }
 
+export function getFederalTaxRates(yearsToInflate: number): Rate[] {
+    const shouldUse2025Rate = yearsToInflate === 0 && now().getFullYear() === 2025;
+    if (shouldUse2025Rate) {
+        return getTaxRates(FEDERAL_CODE).map(
+            (rate, index) => ({ ...rate, RATE: index === 0 ? CA_LOWEST_TAX_RATE_2025 : rate.RATE }),
+        );
+    }
+    return getTaxRates(FEDERAL_CODE);
```

Comment:
Oui c'est redondant de faire de la dé-construction, mais j'ai essayé avec ceci et les tests jest échouaient toujours en disant que le rate était mauvais. Comme si le fait de modifier le rate[0].RATE modifiait l'objet même pour les appels subséquents à la fonction, je n'y comprenais rien pour vrai.
```suggestion
    const shouldUse2025Rate = yearsToInflate === 0 && now().getFullYear() === 2025;
    const rates = getTaxRates(FEDERAL_CODE);
    if (shouldUse2025Rate) {
        rates[0].RATE = CA_LOWEST_TAX_RATE_2025;
    }
    return rates;
```